tests/unittests/CoreGraphics.drawing/ImageComparison.cpp

```diff
@@ -219,6 +219,30 @@ Pixel PixelComparisonModeMask<FailureThreshold>::ComparePixels(const LP& backgro
     return gp;
 }
 
+template <size_t FailureThreshold>
+template <typename LP, typename RP>
+Pixel PixelComparisonModeVisual<FailureThreshold>::ComparePixels(const LP& background, const LP& bp, const RP& cp, size_t& npxchg) {
```

Comment:
I feel that this name does not effectively convey what it actually checks for. 'Perceptible', 'HumanPerceptible', 'NoticeableDifference', or even 'Limen'/'DifferenceLimen' (if you want to be extra nerdy) would be more descriptive.